.github/workflows/pull-request.yml

```diff
@@ -27,81 +27,48 @@ env:
 
 jobs:
 
-  supported-reqs:
+  build-basic:
 
-    name: 'Supported scripts/requirements.txt'
+    name: "build with requirements.txt"
     runs-on: ubuntu-20.04
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: apt-get install
-        run: sudo apt-get -y install $ubuntu_base_deps
-
-      - name: PATH += .local/bin
-        run: echo "$HOME/.local/bin" >> $GITHUB_PATH
-
-      - name: 'pip install -r scripts/requirements.txt'
-        run: pip install -r scripts/requirements.txt
-
-      # No YAML anchors; this is copy/paste
-      - name: configure SOF doc
+      - name: setup env with requirements.txt
         run: |
-          git clone https://github.com/thesofproject/sof
-          cmake -GNinja -S sof/doc -B _build_doxy
+          sudo apt-get -y install $ubuntu_base_deps
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+          pip install -r scripts/requirements.txt
```

Comment:
Collapsing all these steps means more time spent looking for an error when something is wrong in a PR. What is the purpose?